From 33b471ed2e360aee457303ba0d3c84e2b8dc7efd Mon Sep 17 00:00:00 2001
From: Meir Levi <mlevi4@marvell.com>
Date: Thu, 24 Dec 2020 10:35:59 -0500
Subject: [PATCH 1066/1921] Documentation: devicetree: bindings:
 marvell-uio-sam documentation

Marvell UIO for Security Acceleration Module (SAM) documentation

Signed-off-by: Meir Levi <mlevi4@marvell.com>
Change-Id: I09fbd41faf03ecfc7f496f04730f65e7b38838d0
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/42776
Tested-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 .../devicetree/bindings/marvell-uio-sam.txt         | 13 +++++++++++++
 1 file changed, 13 insertions(+)
 create mode 100644 Documentation/devicetree/bindings/marvell-uio-sam.txt

diff --git a/Documentation/devicetree/bindings/marvell-uio-sam.txt b/Documentation/devicetree/bindings/marvell-uio-sam.txt
new file mode 100644
index 000000000000..47a7cfb72ef0
--- /dev/null
+++ b/Documentation/devicetree/bindings/marvell-uio-sam.txt
@@ -0,0 +1,13 @@
+* Marvell UIO for Security Acceleration Module (SAM)
+
+Required properties:
+
+- compatible: should be "marvell,uio-sam".
+- eip_access: phandler to EIP crypto driver
+
+Example:
+
+CP110_LABEL(uio_sam): uio_sam@800000 {
+	compatible = "marvell,uio-sam";
+	eip_access = <&CP110_LABEL(crypto)>;
+};
-- 
2.31.1

